files/matreshka/update.json

```diff
@@ -3,6 +3,13 @@
   "name": "matreshka",
   "repo": "matreshkajs/matreshka",
   "files": {
-    "include": ["matreshka.min.js", "matreshka.js", "matreshka.min.map"]
+    "include": [
+		"./matreshka.min.js",
```

Comment:
Just `matreshka.min.js` here. `./` is only needed for folders where you need to keep the structure